modules/dop/providers/projectpipeline/project_source.go

```diff
@@ -91,6 +101,10 @@ func makeRemote(app *apistructs.ApplicationDTO) string {
 	return filepath.Join(app.OrgName, app.ProjectName, app.Name)
 }
 
+func makeLocation(app *apistructs.ApplicationDTO, t PipelineType) string {
+	return filepath.Join(t.String(), app.OrgName, app.ProjectName)
+}
+
```

Comment:
the `makeLocation` func should not exist in project_source.go

Reply:
done